public/app/features/plugins/admin/api.ts

```diff
@@ -45,14 +47,14 @@ export async function getRemotePlugins(): Promise<RemotePlugin[]> {
 }
 
 async function getPlugin(slug: string): Promise<PluginDetails> {
-  const installed = await getLocalPlugins();
+  const remote = await getRemotePlugin(slug);
+  const isPublished = Boolean(remote);
 
-  const localPlugin = installed?.find((plugin: LocalPlugin) => {
+  const [localPlugins, versions] = await Promise.all([getLocalPlugins(), getPluginVersions(slug, isPublished)]);
```

Comment:
**Suggestion:** I think we can probably remove the fetching of the versions here

As far as I can see we are only using the `getPlugin()` function at [this line](https://github.com/grafana/grafana/blob/0564af2fdd81f9c2a90f276f373d4d8aadf34792/public/app/features/plugins/admin/api.ts#L17), and this is only using the `remote` and `local` properties from the return value.

Reply:
Ah, nice! Even better 🙌🏻